Saule/Http/JsonApiProcessor.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Web.Http.Filters;
+using Saule.Serialization;
+
+namespace Saule.Http
+{
+    /// <summary>
+    /// Processes JSON API responses
+    /// </summary>
+    public class JsonApiProcessor
+    {
+        /// <summary>
+        /// process request as JSON API response
+        /// </summary>
+        /// <param name="context">context</param>
+        public static void ProcessRequest(HttpActionExecutedContext context)
```

Comment:
Could this take the `HttpRequestMessage` and `HttpResponseMessage` (or similar) as parameters? The `ActionExecutedContext` is very closely tied to action filters.